arelle/plugin/validate/ESEF/Util.py

```diff
@@ -122,7 +125,10 @@ def resourcesFilePath(modelManager: ModelManager, fileName: str) -> str:
     return os.path.join(_resourcesDir, fileName)
 
 def loadAuthorityValidations(modelXbrl: ModelXbrl) -> list[Any] | dict[Any, Any]:
-    _file = openFileStream(modelXbrl.modelManager.cntlr, resourcesFilePath(modelXbrl.modelManager, "authority-validations.json"), 'rt', encoding='utf-8')  # type: ignore[no-untyped-call]
+    _file = openFileStream(modelXbrl.modelManager.cntlr, resourcesFilePath(modelXbrl.modelManager, "authority-validations.json"), 'rt', encoding='utf-8')
+    assert not isinstance(_file, io.IOBase)
+    assert not isinstance(_file, FileSource)
```

Comment:
`json.load` requires `SupportsRead[str | bytes]` but `close()` is a method of `io.IOBase`, hence the asserts here.